scalalib/api/src/mill/scalalib/api/ZincWorkerUtil.scala

```diff
@@ -65,9 +65,8 @@ trait ZincWorkerUtil {
     case _ => scalaVersion
   }
 
-  private case class Version(major: Int, minor: Int)
-  private def minorMajorVersion(version: String): Version = version match {
-    case PartialVersion(major, minor) => Version(major = major.toInt, minor = minor.toInt)
+  private def binaryVersion(version: String): ( /*major: */ Int, /*minor:*/ Int) = version match {
+    case PartialVersion(major, minor) => (major.toInt, minor.toInt)
```

Comment:
I wonder what binary issue that may be, since the case class is `private`.

Reply:
It should be effectively safe, but maybe there is either a false positive or some possible incompatibility when defining nested classes inside traits. 

https://github.com/com-lihaoyi/mill/actions/runs/17394751012/job/49374829543#step:4:308
```
[7629] Found 1 issue when checking against com.lihaoyi:mill-scalalib-api_2.13:0.11.4
[7629]  * abstract synthetic method mill$scalalib$api$ZincWorkerUtil$$Version()mill.scalalib.api.ZincWorkerUtil#Version# in interface mill.scalalib.api.ZincWorkerUtil is present only in current version
   filter with: ProblemFilter.exclude[ReversedMissingMethodProblem]("mill.scalalib.api.ZincWorkerUtil.mill$scalalib$api$ZincWorkerUtil$$Version")
```

Reply:
Seems like it would fix it. Should I replace the current tuple variant with the private[this] alternative? 
I'm considering keeping the tuple since the `PartialVersion` would clash with existing regex, and `Version` is too opaque

Reply:
I've made a mistake and tested mima on `scalalib` instead of `scalalib.api` subproject. It still fails when using `private[this]`, I've adjusted mima filter since that should be safe change